_lib/wordpress_featured-topic_processor.py

```diff
@@ -0,0 +1,51 @@
+import sys
```

Comment:
Even though we may not be importing this module directly, the dash in the filename makes me uneasy.  Can we change to underscore?